projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -655,6 +657,56 @@ public function pre_comment_on_post() {
 		}
 	}
 
+	/**
+	 * Handle Jetpack Comments POST requests: process the comment form, then client-side POST the results to the self-hosted blog
+	 *
+	 * This function exists because when we submit the form via the jetpack.wordpress.com iframe
+	 * in Chrome the request comes in to jetpack but for some reason the request doesn't hace access to cookies yet.
+	 * By submitting the form again locally with the same data the process works as expected.
+	 *
+	 * @return never
+	 */
+	public function retry_submit_comment_form_locally() {
+		// We are not doing any valiodation here since all the validation will be done again by pre_comment_on_post().
+		// if the comment has been posted, this is where we do our service verification checks
+		$comment_data = stripslashes_deep( $_POST ); // phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
Is the comment above this line related specifically to `$comment_data = stripslashes_deep( $_POST );`, and in response to the PHPCS ignore comment?

Generally in the monorepo I see we add the comment in-line after the PHPCS ignore in that case, and possibly because of that I was slightly confused as to what the above comment referenced.

If the comment above is meant to cover the whole function then it could be worth just adding '-- verification done in `pre_comment_on_post` - see comment above' or similar. Otherwise I'd add the information in-line.

Also there is a typo - `valiodation` - if keeping the above comment :)

Reply:
The comment was mean to help future devs figure out why we are not doing any validation in this function. 

I simplified it a bit.